tests/unit_tests/test_authentification.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+from django.core import mail
+from django.test import Client
+from pytest_django.asserts import assertRedirects, assertTemplateUsed
+from custom_auth.models import User
+
+client = Client()
+
+@pytest.fixture
+def user_fixture(db, django_user_model):
+    default_user = django_user_model.objects.create(
+        first_name="Alice",
+        last_name="Nolastname",
+        email="alice@wonderland.com",
+        password="nfpt30x49q2obt9"
+    )
+    return default_user
+
+@pytest.mark.django_db
+def test_register():
+    response = client.post(
+        "/accounts/register/",
+        dict(email="rabbit.white@wonderland.com",
+        last_name="Rabbit", first_name="White",
+        password1="FNEYghfr",
+        password2="FNEYghfr"),
+        format="text/html")
+    assert response.status_code == 404
```

Comment:
I'm not sure why you are testing that this page does not exist... There are a lot of urls that would give us a 404